store/CSDK/CarbonWriter.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "CarbonWriter.h"
+
+jobject CarbonWriter::builder(JNIEnv *env) {
+    jniEnv = env;
+    carbonWriter = env->FindClass("org/apache/carbondata/sdk/file/CarbonWriter");
+    jmethodID carbonWriterBuilderID = env->GetStaticMethodID(carbonWriter, "builder",
+        "()Lorg/apache/carbondata/sdk/file/CarbonWriterBuilder;");
+    carbonWriterBuilderObject = env->CallStaticObjectMethod(carbonWriter, carbonWriterBuilderID);
+    return carbonWriterBuilderObject;
+}
+
+jobject CarbonWriter::outputPath(char *path) {
+    jclass carbonWriterBuilderClass = jniEnv->GetObjectClass(carbonWriterBuilderObject);
+    jmethodID carbonWriterBuilderID = jniEnv->GetMethodID(carbonWriterBuilderClass, "outputPath",
+        "(Ljava/lang/String;)Lorg/apache/carbondata/sdk/file/CarbonWriterBuilder;");
+    jstring jPath = jniEnv->NewStringUTF(path);
+    jvalue args[1];
+    args[0].l = jPath;
+    carbonWriterBuilderObject = jniEnv->CallObjectMethodA(carbonWriterBuilderObject, carbonWriterBuilderID, args);
+    return carbonWriterBuilderObject;
+}
+
+jobject CarbonWriter::withCsvInput(char *jsonSchema) {
+
+    jclass carbonWriterBuilderClass = jniEnv->GetObjectClass(carbonWriterBuilderObject);
+    jmethodID carbonWriterBuilderID = jniEnv->GetMethodID(carbonWriterBuilderClass, "withCsvInput",
```

Comment:
just rename these variables as methodID

Reply:
it's based on PR2792, not in this PR, changed to id